src/usr/local/www/system_usermanager_settings.php

```diff
@@ -199,6 +205,15 @@
 	$auth_servers
 ));
 
+$section->addInput(new Form_Input(
+        'auth_refresh_time',
+        'Auth Refresh Time',
+        'number',
+        $pconfig['auth_refresh_time'],
+        ['min' => 0, 'max' => 3600]
+))->setHelp('Time in seconds to cache authentication results. The default is 30 seconds, maximum 3600 (one hour). '.
```

Comment:
It should be checked before this page saves the settings, similar to the validation performed on the session timeout value: https://github.com/scherma/pfsense/blob/c6aca57dcfa8d1f5d19c1c50a1ef8fec3e2aba1f/src/usr/local/www/system_usermanager_settings.php#L114